### PR TITLE
select Notification From Notification Drawer

### PR DESCRIPTION
**Pull request by DarshanS4444**
Added Implementation for selectNotificationFromNotificationDrawer() method
Added Scenario to test the feature for JioMeet application - Meeting Notification

### CODE REVIEW

File: src/main/java/com/znsio/teswiz/runner/Driver.java [hunk 1]
```diff
@@ -287,19 +290,22 @@ public void openNotifications() {
...
         AppiumDriver appiumDriver = (AppiumDriver) this.driver;
+        Dimension screenSize = appiumDriver.manage().window().getSize();
+        PointerInput touch = new PointerInput(PointerInput.Kind.TOUCH, "touch");
+        Sequence dragNotificationBar = new Sequence(touch, 1);
+        dragNotificationBar.addAction(touch.createPointerMove(Duration.ofSeconds(0), PointerInput.Origin.viewport(), screenSize.width / 2, 0));
+        dragNotificationBar.addAction(touch.createPointerDown(PointerInput.MouseButton.LEFT.asArg()));
+        dragNotificationBar.addAction(touch.createPointerMove(Duration.ofSeconds(1), PointerInput.Origin.viewport(), screenSize.width / 2, screenSize.height));
+        dragNotificationBar.addAction(touch.createPointerUp(PointerInput.MouseButton.LEFT.asArg()));
+        appiumDriver.perform(singletonList(dragNotificationBar));
+        appiumDriver.perform(singletonList(dragNotificationBar));
```
Comment (anandbagmar) [inline src/main/java/com/znsio/teswiz/runner/Driver.java:303]: Why do we need to perform this action twice?

File: src/main/java/com/znsio/teswiz/runner/Driver.java [hunk 2]
```diff
@@ -287,19 +290,22 @@ public void openNotifications() {
...
 
-    public void selectNotification(By selectNotificationLocator) {
+    public void selectNotificationFromNotificationDrawer(By selectNotificationLocator) {
         AppiumDriver appiumDriver = (AppiumDriver) this.driver;
+        Dimension screenSize = appiumDriver.manage().window().getSize();
+        PointerInput touch = new PointerInput(PointerInput.Kind.TOUCH, "touch");
+        Sequence dragNotificationBar = new Sequence(touch, 1);
+        dragNotificationBar.addAction(touch.createPointerMove(Duration.ofSeconds(0), PointerInput.Origin.viewport(), screenSize.width / 2, 0));
+        dragNotificationBar.addAction(touch.createPointerDown(PointerInput.MouseButton.LEFT.asArg()));
+        dragNotificationBar.addAction(touch.createPointerMove(Duration.ofSeconds(1), PointerInput.Origin.viewport(), screenSize.width / 2, screenSize.height));
```
Comment (anandbagmar) [inline src/main/java/com/znsio/teswiz/runner/Driver.java:300]: How far are we dragging the notification drawer?

File: src/test/resources/com/znsio/teswiz/features/jiomeet.feature [hunk 3]
```diff
@@ -45,4 +45,11 @@ Feature: In a meeting scenarios
     And "Guest-2" joins the meeting from "jiomeet-chrome" on "web"
     Then "Host" should be able to get to chat window
     When "Host" sends "Hey" chat message
-    Then "Guest-2" should see the chat message on its chat window
\ No newline at end of file
+    Then "Guest-2" should see the chat message on its chat window
+
+#  CONFIG=./configs/jiomeet_local_config.properties TAG="@jiomeet and  @selectNotificationTest" PLATFORM=android ./gradlew run
+    @selectNotificationTest @android
+    Scenario: User can select Notification
```
Comment (snehith07) [inline src/test/resources/com/znsio/teswiz/features/jiomeet.feature:52]: does this seem better?
"user can view and tap on notification bar"
Or
"validating tap on the notification from notification drawer"

File: src/main/java/com/znsio/teswiz/runner/Driver.java (hunk 1 above)
Comment (DarshanS4444) [reply src/main/java/com/znsio/teswiz/runner/Driver.java:303]: First action is to highlight the notification bar as it won't be highlighted. 
Second action is to drag the notification bar.

File: src/main/java/com/znsio/teswiz/runner/Driver.java (hunk 2 above)
Comment (DarshanS4444) [reply src/main/java/com/znsio/teswiz/runner/Driver.java:300]: we are dragging the bar completely until screen width

File: src/test/resources/com/znsio/teswiz/features/jiomeet.feature (hunk 3 above)
Comment (DarshanS4444) [reply src/test/resources/com/znsio/teswiz/features/jiomeet.feature]: updated the scenario name

Comment (snehith07) [reply src/test/resources/com/znsio/teswiz/features/jiomeet.feature]: is there any change in the scenario name?

File: caps/jiomeet_local_capabilities.json [hunk 4]
```diff
@@ -1,9 +1,7 @@
 {
   "android": {
     "adbExecTimeout": 45000,
-    "app": {
-      "local": "https://github.com/anandbagmar/sampleAppsForNativeMobileAutomation/raw/main/com.jio.rilconferences_v4.21.6.7-421607_Android-5.0.apk"
-    },
+    "app": "src/test/resources/sampleApps/jioMeet.apk",
```
Comment (anandbagmar) [inline caps/jiomeet_local_capabilities.json:4]: why was the name of the apk changed?

**Pull request merged**